### PR TITLE
List jobs page styling

### PR DESCRIPTION
**Pull request by jewelltaylor**
# PR Type
[ **Feature** | Fix | Documentation | Other ]

# Short Description

Clickup Ticket(s): [List Jobs Page Styling](https://app.clickup.com/t/868899xqp)
Small PR to as a follow up to #21 to add template-based styling to the list jobs page. Specifically, this involved: 
- Adapting the styling of the tables use to store jobs in each status
- Add a single loading gif that renders while data is being fetched
- Add highlighting of navbar items based on currently selected page. Also add effect on hover of navbar item. 

Here is an example of the page currently with example data:
image

As mentioned, one of the drawbacks of my implementation of the single loading gif is that I have to do the fetching in a higher level component so that I can render the loading gif if any of the fetches isLoading. Since I can't put hooks inside a callback, I have to hardcode the fetches in a list. If you have any ideas how to get around this, or other suggestions I would appreciate them :) 

# Tests Added
Minor adaptation of current tests to pass with new ui. Let me know if you think I should add any additional tests :)

### CODE REVIEW

**Review by lotif — COMMENTED** [review]
It's looking great! Just a couple of small comments. I like your solution for loading everything at the top level component, I had another solution in mind but it would be more cumbersome so let's stick with this one!

File: florist/app/jobs/page.tsx [hunk 1]
```diff
@@ -69,26 +107,39 @@ export function StatusTable({
...
         return (
-            <div>
-                <span data-testid={`status-no-jobs-${status}`}>
-                    {" "}
-                    No jobs to display.{" "}
-                </span>
+            <div className="card-body px-0 pb-2">
+                <div className="ps-3">
+                    <span data-testid={`status-no-jobs-${status}`}>
+                        {" "}
```
[the remark this answers is not in the document]
Comment (jewelltaylor) [reply florist/app/jobs/page.tsx]: Don't know how that got in there! I removed :)

File: florist/tests/unit/app/jobs/page.test.tsx [hunk 2]
```diff
@@ -56,7 +56,7 @@ describe("List Jobs Page", () => {
         const { container } = render(<Page />);
         const h1 = container.querySelector("h1");
         expect(h1).toBeInTheDocument();
-        expect(h1).toHaveTextContent("Jobs By Status");
+        expect(h1).toHaveTextContent("Jobs");
```
[the remark this answers is not in the document]
Comment (jewelltaylor) [reply florist/tests/unit/app/jobs/page.test.tsx:59]: Good idea! Let me know if the tests I added are sufficient

File: florist/tests/unit/app/jobs/page.test.tsx [hunk 3]
```diff
@@ -109,4 +109,37 @@ describe("List Jobs Page", () => {
...
+            [],
+            false,
+            true,
+        );
+        const { getByTestId } = render(<Page />);
+        const element = getByTestId("jobs-page-loading-gif");
+        expect(element).toBeInTheDocument();
+    });
+
+    it("Doesnt Render Loading GIF when no Loading", () => {
```
Comment (lotif) [inline florist/tests/unit/app/jobs/page.test.tsx:129]: Typos on "Does't" and "not"

**Pull request merged**